A3-channel-tracing.md

```diff
@@ -43,6 +43,17 @@ _Trace events_ should only be added for events that happen relatively infrequent
 ... Entering idle mode
 ```
 
+## Minimal Set of Data
+
+We define a minimal set of events that must be traced, in order to consider the channel tracing feature as complete.
+
+All of these events must be traced:
+- Channel creation and deletion
+- Subchannel creation and deletion
+- Address resolution
```

Comment:
BTW, as an implementation detail, it's probably a good idea to make sure that LB policies have the ability to add trace events, since there may be policy-specific events of interest.  For example, the grpclb policy may want to add an event when it establishes or loses a connection to a balancer, or when it receives a serverlist with all drop entries, or something else.